pkgs/development/python-modules/ipywidgets/default.nix

```diff
@@ -1,48 +1,40 @@
-{ lib
-, buildPythonPackage
+{ buildPythonPackage
 , fetchPypi
-, python
-, nose
-, pytest
-, mock
-, ipython
 , ipykernel
+, ipython_genutils
 , jupyterlab-widgets
+, lib
+, nbformat
+, pytestCheckHook
 , traitlets
-, notebook
 , widgetsnbextension
 }:
 
 buildPythonPackage rec {
   pname = "ipywidgets";
   version = "7.7.0";
+  format = "setuptools";
```

Comment:
Why? Can we not switch to pyproject right away?

Reply:
Good question... there's a pyproject.toml but it appears to only have linting/typechecking rules in it, so we're stuck with setuptools for the time being unless I'm mistaken.